src/Pages/Main/Main.scss

```
@@ -0,0 +1,176 @@
+.Main {
+  position: relative;
+  width: auto;
+  height: auto;
+
+  .videologo-continer {
+    position: relative;
+    width: 100%;
+    height: auto;
+    .mainvideo {
+      position: relative;
+      width: 100%;
+      height: auto;
+    }
+    .unfolding-logo {
+      position: absolute;
+      background-color: black;
+      color: white;
+      padding: 20px;
+      font-size: 20px;
+      z-index: 4;
+      bottom: 4px;
+      left: 0;
+
+      &:hover {
+        background-color: gray;
+      }
+    }
+  }
+
+  .shop {
+    width: 1130px;
+    height: 33px;
+    font-size: 40px;
+    font-weight: 300;
+    margin: 0 auto;
+    margin-top: 30px;
+    margin-bottom: 40px;
+  }
+  .categorycontainer {
+    display: flex;
+    width: 1170px;
+    height: 362.688;
+    margin: 0 auto;
+    justify-content: space-between;
+    .main-categorys {
+      width: 1120px;
```

```suggestion
    margin-bottom: 40px;
  }
  
  .categorycontainer {
    display: flex;
    width: 1170px;
    height: 362.688;
    margin: 0 auto;
    justify-content: space-between;
    
    .main-categorys {
      width: 1120px;
```
- 여기도 마찬가지로 새로운 셀렉터 앞에서는 한줄씩 띄워 주시고 카멜케이스로 클래스명 작성해주세요
- 클래스명 오타라서 mainCategories로 수정해주세요